### DISCUSSION TITLE
Self hosted supabase Oauth Redirect is incorrect.

### DISCUSSION THREAD

**Discussion by jahzlariosa** [supabase/supabase#26874] [Questions] [answered]
I tried setting up self hosted supabase using coolify
Everything works.

Problem is that in my nextjs Oauth.
I was able to make everything work except that after authentication instead getting redirected to my app https://localhost:3000/?code=XXXXXXXX I'm getting redirected to https://my-selfhosted-supabase.com/?code=XXXXXXXX

Note everything works well if I use supabase itself.

**Comment by jahzlariosa** [comment, marked as answer]
Figured it out :D 

forgot to setup ADDITIONAL_REDIRECT_URLS

**Reply by rRakarare** [reply]
Could you explain where you added "ADDITIONAL_REDIRECT_URLS" and where to redirect ?. I am having the same problem using nextjs and i am a bit confused of redirect_urls. Thanks :)
